MobilityData/mobility-feed-api
# feat: Collecting data from API registration workflow

**Summary:**

We need to store the contact data collected in the form that users see [when they register for an API account](https://www.figma.com/file/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?type=design&node-id=165-95&mode=dev).

Closes #113 
![image](https://github.com/MobilityData/mobility-feed-api/assets/60586858/50476ffe-932d-4252-9981-4b6269b07d84)

**Expected behavior:** 
- Before the user registers: the registration page is displayed 
- Once the user is registered: should be redirected to 

**Testing tips:**
Using any sign up/login method follow the following steps:
- Sign up 
- Complete the registration process (you should be redirected to the registration page)
- Upon completion you should be redirected to the Account page (no action required on your part)
- Visit [the datastore console](https://console.cloud.google.com/datastore/databases/-default-/entities;kind=web_api_users;ns=__$DEFAULT$__/query/kind?project=mobility-feeds-dev&lastRefresh=1701276748158) to see that your information was registered.
- Sign out
- Sign in again (you should directly be redirected to the account page with your registration information visible)

*Note 1*: If you sign in with github and your email is hidden, you should not have access to the option to register to email announcements from the registration page
*Note 2*: If you register to email announcements, you should be able to see it in your account page as displayed bellow.
![image](https://github.com/MobilityData/mobility-feed-api/assets/60586858/af46b6b3-76d6-46bf-a18a-7c6fcff1ae1c)


Please make sure these boxes are checked before submitting your pull request - thanks!
